vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ExhibitionManager.java

```diff
@@ -66,4 +69,13 @@ public IExhibition getStartExhibition() {
         }
         return null;
     }
+    
+    @Override
+    public IExhibition addPreviewId() {
```

Comment:
this method doesn't just add a preview id, it creates an exhibition object. The method name needs to reflect hat!

also, needs javadoc